tools/mkmapdata/mkmapdata.sh

```diff
@@ -346,7 +337,7 @@ case $hostname in
   intelvers=17.0.1
   module load esmf_libs/$esmfvers
   module load intel/$intelvers
-  module load ncl
+  module load ncl/6.6.2
```

Comment:
Is there a need for a specific version of NCL? If not using the generic one seems better.

Reply:
`module load ncl` returns this error:
Lmod has detected the following error:  These module(s) exist but cannot be loaded as requested: "ncl"
   Try: "module spider ncl" to see how to load the module(s).

...while `module load ncl/6.6.2` works.

Reply:
I had to load intel/18.0.5 recently to get the generic ncl to work with mksurfdata_map.

`intelvers=18.0.5`

Reply:
I was just going to say the same thing, if you use intelvers=18.0.5 you can load generic ncl, but otherwise you have to name the version. And you should do the update to intel 18 anyway, as we have that on the release branch.

Reply:
I think it's a bug on CISL's part that you can't load ncl without naming a specific version. We could create a ticket for them about this.